src/Containers/FeaturedProfilesCard.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import styled from 'styled-components';
+import { AddUser } from '@styled-icons/entypo/AddUser';
+import { flex } from '../Utils/Mixins';
+
+export interface IProfile {
+    image: string;
+    initials: string;
+    id: number;
+}
+
+export interface IFeaturedProfilesCardProps {
+    profileData: IProfile[];
+    alt: string;
+    width?: number;
+    height?: number;
+}
+
+export const FeaturedProfilesCard: React.FC<IFeaturedProfilesCardProps> = ({
+    alt,
+    profileData,
+    width = 100,
+    height = 100,
+}): React.ReactElement => {
+    const getProfileCircles = (): any => {
+        const PROFILE_PICTURE_LIMIT = 3;
```

Comment:
this has nothing to do with the componet it should be at top of the file

Reply:
Does this mean the getProfilesCircles function does not need to be in the FeaturedProfilesCard component? I currently have getProfileCircles accessing props from the component.

Reply:
I mean the variable 3 does not need to be inside the function